code/modules/mob/living/carbon/human/species/necromorph/exploder.dm

```diff
@@ -206,7 +206,7 @@ The last resort. The exploder screams and shakes violently for 3 seconds, before
 	//A normal explosion
 	spawn()
 		//Max power 2 because hull breaches are not cool
-		T.explosion(5,2)
+		T.explosion(10,2)
```

Comment:
this should have a max radius of 5, thats already pretty large

Reply:
I tested it in game and it felt a little small, but sure